src/server/game/Miscellaneous/SharedDefines.h

```diff
@@ -6508,4 +6508,19 @@ enum class GameError : uint32
     ERR_NOT_IN_PET_BATTLE                                           = 1047,
 };
 
+enum class MountResult : uint32
+{
+    InvalidMountee = 0,
+    TooFarAway     = 1,
+    AlreadyMounted = 2,
+    NotMountable   = 3,
+    NotYourPet     = 4,
```

Comment:
In what cases this used?

Reply:
Maybe parked mounts without vehicle access?